Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Skin Shader Without Decals

## Developer changelog:
- Created `portrait_skin_attachment_alpha_to_coverage` shader that doesn't apply decals.


# How to test:
Check the draenei horns, check they have no issues and look like they did before.
